packages/@coorpacademy-nova-icons/scripts/generate-icons.js

```diff
@@ -43,7 +43,11 @@ type JSXElement = {|
 const isFillAttribute = ({name: {name}, value: {value} = {}}: JSXAttribute): boolean =>
   name === 'fill' && value !== 'none';
 
-const replaceWithPropValue = types => types.jsxExpressionContainer(types.identifier('props.color'));
+const isStrokeAttribute = ({name: {name}, value: {value} = {}}: JSXAttribute): boolean =>
```

Comment:
in english we should say : "HASstrokeAttribute" intead of "IS strokeAttribute". It sounds more correct

Reply:
I'm testing the attribute, not the parent having attribute. I'm just checking that the attribute is a `stroke` attribute (or `fill` above)

Reply:
Sorry i'll dismiss the review for this reason